.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us triage the problem and find a fix.
+title: "[Bug]"
+labels: bug, needs triage
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+etc.
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Suspect component/Location where the bug might be occuring**
+Please provide this if you know where this bug might occur otherwise leave as `unknown`
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Runtime (please complete the following information):**
+ - OS: [e.g. iOS]
```

Comment:
Is `iOS` a good example here? Linux or Mac would make more sense since operators usually run on those OSes.

Reply:
Noted, will change, Left the default that github suggested